build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1890,6 +1890,71 @@ message DigestFunction {
     // The BLAKE3 hash function.
     // See https://github.com/BLAKE3-team/BLAKE3.
     BLAKE3 = 9;
+
+    // Use git-sha1 hahsing function prefixed with a one byte marker.
+    //
+    // Since git hashes blob content differently from trees, this type of
+    // information has to be transmitted in addition to the content and the
+    // hash. To this aim, the git hash values passed over the wire are prefixed
+    // with a single-byte marker, thus allowing the remote side to distinguish a
+    // blob from a tree without inspecting the (potentially large) content. The
+    // markers are
+    //
+    // - 0x62 for a git blob (0x62 corresponds to the character b)
```

Comment:
Which marker is used for Action and Command messages? Are those treated as plain blobs?

Reply:
Sorry, maybe I was unclear. They get a marker. Action, for example is a blob and so will be marked with 62.

Everything is consistenly hashed according to the git-sha1 (which is different from plain sha1). If the object is a blob the hash is prefixed with `0x62`, if it is a tree (for example the input root), with `0x74`

Did I add even more entropy? :)